topics/statistics/tutorials/clustering-machinelearning/tutorial.md

```diff
@@ -0,0 +1,482 @@
+---
+layout: tutorial_hands_on
+
+title: 'Clustering in Machine Learning'
+zenodo_link: https://zenodo.org/...
+questions:
+- How to use clustering methods to categorized data in different clusters
+objectives:
+- Learn clustering background
+- Learn hierarchical clustering algorithm
+- Learn k-means clustering algorithm
+- Learn DBSCAN clustering algorithm
+- Apply clustering algorithms to different data sets
```

Comment:
Clustering methods or clustering algorithms? Please maintain consistency

Reply:
Done! all are changed to clustering algorithms